fhir-server/src/main/java/com/ibm/fhir/server/rest/FHIRRestInteractionVisitorPersist.java

```diff
@@ -300,6 +302,21 @@ private void doInteraction(int entryIndex, String requestDescription, long initi
                         .build())
                     .build();
             setEntryComplete(entryIndex, entry, requestDescription, initialTime);
+        } catch (FHIRPersistenceIfNoneMatchException e) {
+            if (failFast) {
+                String msg = "Error while processing request bundle.";
+                throw new FHIRRestBundledRequestException(msg, e).withIssue(e.getIssues());
+            }
```

Comment:
I updated some very similar code in the past to include the entryIndex in both the error message and the issue...maybe we should do the same in each catch block here?

Reply:
https://github.com/IBM/FHIR/pull/2960/files#diff-35b7e53e0ed72c5da33f937ee4634c89bb018a496a8c2db4e220a75117c59507R396

Reply:
Yep, I'll do that